docs/articles/discovery/index.md

```diff
@@ -1,4 +1,9 @@
-# Discovery Overview
+---
+uid: akka-discovery
```

Comment:
Needed to add a UID here in order to use `xref` links, per https://getakka.net/community/contributing/documentation-guidelines.html#all-links-between-documents-must-use-xref